systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -297,6 +300,73 @@ public KafkaBuilder defaultKafka(String name, int kafkaReplicas) {
                     .endSpec();
     }
 
+    DoneableKafka kafkaJBOD(String name, int kafkaReplicas, int diskCount, int diskSizeGi, boolean deleteClaim) {
+        return kafka(jbodKafka(name, kafkaReplicas, diskCount, diskSizeGi, deleteClaim).build());
```

Comment:
I think better approach there is to call `defaultKafka()` and edit spec according parameters.